sf-docker/sf_dependencies/docker-compose.yml

```diff
@@ -8,13 +8,21 @@ services:
     environment: 
       - ACCEPT_EULA=Y
       - SA_PASSWORD=Test1234
+    restart: unless-stopped
```

Comment:
If we are doing this for the dependencies, we could also set it up for SF

Reply:
added